static/usr/bin/splash-client

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -exu
+
+# Change 1 is "Initialize system state"
+msg=
+prev_msg=
+while true; do
+    sleep 0.5
+    msg=$(snap change --abs-time 1 | grep -m1 Doing |
+              awk -F ' +' '{ s = ""; for (i = 4; i <= NF; i++) s = s $i " "; print s }') \
```

Comment:
This is fine. But here are some alternatives:

Without awk:
```
tr -s " " | cut -d" " -f4-
```

Merging awk and grep together, and removing the loop (which also does not squash multiple spaces):
```
awk -F " +" '$1 == "Doing" { print substr($0, index($0,$4)); exit }'
```

Reply:
And for the lol in sed:
```
sed -E '/^Doing /{;s/([^ ]*[ ]*){3}//;q};d'
```